### ISSUE TITLE
Option to disable certificate checks

### ISSUE THREAD

**Issue by go-rev** [saferwall/pe#58]
Similar to  #52.
Can you please add an option to disable cert related  functions through options?
One of the issue with this is the launching of certutil (which will be flagged by many EDRs). Other issue is, when a program using this lib is run for the first time on a system without network (like an analysis VM), it takes ~2 minutes to process a PE file with signatures

**Comment by LordNoteworthy** [comment]
@go-rev 

Thanks a lot for reporting this !

I do agree with all of what you said. 

1. I can definitely replace using `certutil`.
2. If a system does not have internet access, I need a way to quickly fail, seems like there is a ~ 2min timeout somewhere which need to be adjusted.

Yes, I can add an option to disable cert validation.

**Comment by LordNoteworthy** [comment]
I just released a tag v1.3.5 that gives an option to disable cert check, by default it is enabled, so you need to supply it like this:

```go
	pe, err := peparser.NewBytes(data, &peparser.Options{
		DisableCertValidation: true,
	})
```

Does it solves your issue ?

**Comment by go-rev** [comment]
Thank you for the quick fix. I can confirm that with the new option, `certutil` is not launched and delay during the first run is gone.
